### PR TITLE
Add resource attributes to the base template (for wb-main, wb-webpage, wb-publisher)

### PR DESCRIPTION
**Pull request by ctopCanada**
WET - 123 Add resource attributes to the base template (for wb-main, wb-webpage, wb-publisher)

### CODE REVIEW

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html [hunk 1]
```diff
@@ -28,7 +28,8 @@ <h2 class="wb-inv">Language selection</h2>
 					</li>
 				</ul>
 			</section>
-			<div class="brand col-xs-9 col-sm-5 col-md-4" property="publisher" typeof="GovernmentOrganization">
+
```
Comment (GormFrank) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html:31]: Remove empty line here.

**Review by GormFrank — CHANGES_REQUESTED** [review]
Also remove the changes on the release notes files (files that contain the word "release" in their name).

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/css/theme.min.css">
 	<noscript><link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/wet-boew/css/noscript.min.css" /></noscript>
 </head>
-<body vocab="http://schema.org/" typeof="WebPage">
+<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html:15]: Missing a resource 
* at line 31 for the publisher info 
* at line 88 for the main element.

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
 	<!-- END - Experimental Bootstrap 4 -->
 
 </head>
-<body vocab="http://schema.org/" typeof="WebPage">
+<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
 
 <header class="header-b-line">
 	<div id="wb-bnr" class="container">
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-4.html:22]: Can you add the RDFa markup, like we have for the bootstrap 3 template, for the publisher area.
Here is line 35 to 40

File: site/layouts/splashpage.hbs [hunk 4]
```diff
@@ -20,7 +20,7 @@
 		{{>headresources}}
 	</head>
 
-	<body class="{{pageclass}}" vocab="http://schema.org/" typeof="WebPage">
+	<body class="{{pageclass}}" vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
```
Comment (duboisp) [inline site/layouts/splashpage.hbs:23]: Can you add the same RDFa attribute as the other page on the main element here at line 38

File: site/layouts/servermessage.hbs [hunk 5]
```diff
@@ -9,10 +9,10 @@
 		<meta name="robots" content="noindex, nofollow, noarchive" />
 		{{>servermessageresources}}
 	</head>
-	<body vocab="http://schema.org/" typeof="WebPage">
+	<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
 		<header role="banner" id="wb-bnr" class="container">
 			<div class="row">
-				<div class="col-sm-6" property="publisher" typeof="GovernmentOrganization">
+				<div class="col-sm-6" property="publisher" resource="#wb-publisher" typeof="GovernmentOrganization">
```
Comment (duboisp) [inline site/layouts/servermessage.hbs:15]: Missing RDFa resource in the main element at line 25

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
@ctopCanada please fix the linting error

File: site/layouts/splashpage.hbs [hunk 6]
```diff
@@ -35,7 +35,7 @@
 			<img class="hide" src="https://www.canada.ca/content/dam/canada/splash/sp-bg-11.jpg" alt="" />
 			<img class="hide" src="https://www.canada.ca/content/dam/canada/splash/sp-bg-12.jpg" alt="" />
 		</div>
-		<main role="main">
+		<main role="main" property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement" class="container">
```
Comment (GormFrank) [inline site/layouts/splashpage.hbs:38]: Remove container class on this one:
```suggestion
		<main role="main" property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement">
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — DISMISSED** [review]
@ctopCanada can you squash all the commits into 1 commits.

This was pre-approved, so after your squash we will be able to merged

**Pull request merged**